hazelcast/src/main/java/com/hazelcast/query/impl/Indexes.java

```diff
@@ -77,6 +77,16 @@ public synchronized Index addOrGetIndex(String attribute, boolean ordered) {
     }
 
     public void clearIndexes() {
+        clearIndexes(false);
+    }
+
+    public void clearIndexes(boolean destroy) {
```

Comment:
Could we have javadoc on both to explain the concept and why we need it. We'll forget in a couple of months ;)